core/app/assets/javascripts/spree.js

```diff
@@ -2,7 +2,6 @@
 function Spree () {}
 
 Spree.ready = function (callback) {
-  jQuery(callback)
```

Comment:
Since we use `turbolinks` now, it's enough to listen to `turbolinks:load` event only. Otherwise, `callback` here will be called multiple times that leads to event listeners defined multiple times, and as a result it leads to bugs such "product has been added to the cart twice".